### DISCUSSION TITLE
Nested Component with Repeatable Field

### DISCUSSION THREAD

**Discussion by ninaninani** [strapi/strapi#7633] [Legacy Topics]
Hello, I managed to create a nested repeatable component once as shown below.
But now when I create a new component from a dynamicZone, there is no option for repeatable type.
![image](https://user-images.githubusercontent.com/5014071/91537783-6e432500-e949-11ea-9847-ca4812908eaf.png)

ListText in image below has a repeatable field and it's showing in "Select a component" dropdown.
Now if I create a new component that has repeatable field in it, it doesn't show on the "Select a component" dropdown.
![image](https://user-images.githubusercontent.com/5014071/91538540-8cf5eb80-e94a-11ea-886c-8409f4b909f9.png)

Can anyone help me how to create nested component with a repeatable field ?

**Comment by lauriejim** [comment]
Hello! Can you please share with us your application because you look at having a complex data structure for your application.
This will help us to help you.
Thank you.

**Comment by soupette** [comment]
@ninaninani for UI reasons we have limited the number of nested components to 2 this is why you cannot add your component.
